elixir/apps/domain/lib/domain/actors.ex

```diff
@@ -46,6 +46,16 @@ defmodule Domain.Actors do
     end
   end
 
+  def list_groups_for(%Actor{} = actor, %Auth.Subject{} = subject, opts \\ []) do
```

Comment:
@AndrewDryga, I realized right as I was putting this PR up that I forgot to add tests for this.  I'll get them added in first thing tomorrow.

Reply:
Ok, tests have been added.